src/UserContainer.java

```diff
@@ -3,28 +3,29 @@
 import java.util.Set;
 
 public class UserContainer {
+    private static Map<String, String> users = new HashMap<String, String>();
 
+    public static void setMap(Map<String, String> setNameUsers) {
+        UserContainer.users = setNameUsers;
+    }
 
-    static Map<String, String> map = new HashMap<String, String>();
-
-    public UserContainer() {
+    public static Map<String, String> getUsers() {
+        return users;
     }
 
-    public static boolean isNameExists(String name, String password) {
-        boolean flag = false;
+    public static boolean isNameAndPasswordExisits(String name, String password) {
+        boolean existsNameAndPassword = false;
```

Comment:
bolean shows if name and password exists.
Does it mean that user exists in the system?
If yes, maybe better to name it like `userExists` ?